### ISSUE TITLE
[Feature Request] Add E-Mail notification for due date

### ISSUE THREAD

**Issue by rabesocke** [nextcloud/deck#1202]
**Is your feature request related to a problem? Please describe.**
We have several people in our club who don't use Nextcloud very often. These guys have to be reminded again and again to check the cards and their due dates.

**Describe the solution you'd like**
It should be possible to get an E-Mail with a list of cards where the due date is near or over.

**Describe alternatives you've considered**
E-Mail notifications via the calendar is also possible (#15), but a direct E-Mail notification from Deck would be better in order to avoid messing up the calendars.

**Additional context**
#566 and #414

**Comment by rabesocke** [comment]
Are there any plans to integrate this feature request?

**Comment by SzOr** [comment]
Perhaps a bounty?

**Comment by rabesocke** [comment]
I have to check, if our club can support this via a bounty.

**Comment by juliushaertl** [comment]
Makes sense to have so adding to develop, but note that this doesn't mean that this is scheduled for development yet.
